processing/src/main/java/io/druid/query/aggregation/last/DoubleLastAggregatorFactory.java

```diff
@@ -136,7 +149,7 @@ public void aggregate(ByteBuffer buf, int position)
             long lastTime = buf.getLong(position);
             if (pair.lhs >= lastTime) {
               buf.putLong(position, pair.lhs);
-              buf.putDouble(position + Long.BYTES, pair.rhs);
+              buf.putDouble(position + Double.BYTES, pair.rhs);
```

Comment:
It really skips a long, so this line shouldn't be changed

Reply:
done.